app/src/androidTest/java/com/group80/uoftinder/MatchInteractorUnitTest.java

```diff
@@ -0,0 +1,167 @@
+package com.group80.uoftinder;
```

Comment:
Quick question did you end up moving the original UpdateListTest tests over here?

Reply:
Yes, that is correct